articles/quickstarts/install-command-line.md

```diff
@@ -96,12 +93,6 @@ To run the application:
 
 Verify your installation by creating a Q# `Hello World` application.
 
-1. Install the project templates.
```

Comment:
To maintain consistency with the VS and VSCode tabs, I took out the step that repeats the install.